### ISSUE TITLE
Add term Cone-shaped distal tibial epiphysis

### ISSUE THREAD

**Issue by lisannesw** [obophenotype/human-phenotype-ontology#10378]
**Preferred term label:**
Cone-shaped distal tibial epiphysis

**Synonyms**

**Definition (free text, please give PubMed ID)**
The distal epiphysis (rounded portion of bone at the far end of the tibia distal to the growth plate) has an abnormal cone-shaped appearance.

**Parent term (use hpo.jax.org/app)**
Cone-shaped epiphysis HP:0010579

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
ACROCAPITOFEMORAL DYSPLASIA; ACFD, OMIM607778

**Your nano-attribution (ORCID)**
Andreas Zankl: 0000-0001-8612-1062, @azankl
Lisanne Wit: 0009-0007-1020-5412

**Comment by pnrobinson** [comment]
@lisannesw 

Adding new term: 
- ID: HP:6000778
- label: Cone-shaped distal tibial epiphysis
- parents: [Abnormality of the distal tibial epiphysis (HP:0010592)](https://hpo.jax.org/app/browse/term/HP:0010592)
- pmid: PMID:12624140
- def: The distal epiphysis (rounded portion of bone at the far end of the tibia distal to the growth plate) has an abnormal cone-shaped appearance, that is, the epiphysis has a globular form at the bottom, tapering to a point at the top.
- comment: 
- orcid: orcid:0000-0001-8612-1062
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10378
- synonyms:
